lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -0,0 +1,42 @@
+<div class="u-field-value field">
+    <% if (editable !== 'never') { %>
+        <% if (title && titleVisible) { %>
+            <label class="u-field-title field-label" for="u-field-select-<%- id %>">
+                <%- title %>
+            </label>
+        <% } else { %>
+            <label class="sr" for="u-field-select-<%- id %>">
+                <%- screenReaderTitle %>
+            </label>
+        <% } %>
+    <% } %>
+
+    <% if (iconName) { %>
+        <span class="u-field-icon icon fa <%- iconName %> fa-fw" aria-hidden="true"></span>
+    <% } %>
+
+    <% if (editable === 'never') { %>
+        <span class="sr"><%- screenReaderTitle %></span>
+        <span class="u-field-value-readonly"></span>
+    <% } else { %>
+        <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (showBlankOption) { %>
+                <option value=""></option>
+            <% } %>
+            <% _.each(selectOptions, function(selectOption) { %>
+                <option value="<%- selectOption[0] %>"><%- selectOption[1] %></option>
```

Comment:
The option element should have a selected (boolean) attribute defined if the option is currently selected. https://www.w3.org/TR/html5/forms.html#attr-option-selected